hs_rest_api/discovery.py

```diff
@@ -27,7 +27,7 @@ class Meta:
             "start_date",
             "end_date",
             "east",
-            "west",
+            "north",
```

Comment:
Is my understanding correct that east and north define a point, and prior to this change, some resources may not have displayed a point, since only east and west were being returned, and not a north value?

Reply:
It was an undiscovered bug in Scott's API.